packages/logger/index.ts

```diff
@@ -31,7 +31,7 @@ type ChildKey =
   | "aila:stream"
   | "aila:rag"
   | "aila:testing"
-  | "aila:chat"
+  | "aila:chat-performance"
```

Comment:
`aila` is for the aila package. Maybe you want `performance` or `ui:performance`?